### ISSUE TITLE
Hoppers & cauldrons fail to render the texture while PBR shaders are enabled in the nether

### ISSUE THREAD

**Issue by Arrowpuppet** [comp500/Indium#101]
![image](https://user-images.githubusercontent.com/43303412/150474829-ef4e379e-3b19-41d0-a2e3-ef02edb61135.png)
Hoppers don't seem to render the textures correctly while in the nether, instead just showing up as a flat texture. This can be solved by removing indium, so I suspect that the issue could be caused in some part by indium. The textures show up properly in every other dimension, so this issue is for whatever reason, exclusive to the nether. Disabling PBR in the shader also fixes the issue, so it's possible that indium causes some conflict with how shaders process PBR in the nether.
If there is any other info I need to provide to help solve this issue, please let me know.

**Comment by comp500** [comment]
I have no idea how PBR works... does it work with just Sodium, Indium and Iris? Indium shouldn't affect anything if there isn't a mod installed that uses the Fabric Rendering API.

**Comment by Arrowpuppet** [comment]
I just tested it, and the textures work fine with just sodium, iris, and indium. Here's the full list of loaded mods; I'm not sure exactly which mods depend on Indium that could be causing this, would you have any ideas?
```	Fabric Mods: 
		advanced_runtime_resource_pack: Runtime Resource Pack 0.5.4
		advancementinfo: AdvancementInfo 1.18-fabric0.43.1-1.3
		advancements-enlarger: Advancements Enlarger 0.2.4
		animatica: Animatica 0.1+1.17
		antighost: AntiGhost 1.18-fabric0.43.1-1.1.4
		appleskin: AppleSkin mc1.18-2.2.0
		architectury: Architectury 3.2.51
		armorhud: Simple armor hud 1.18-1.2.1
		authme: Auth Me 2.2.0
		autoconfig1u: Auto Config v1 Updated 3.2.2
		bedspreads: Bedspreads 1.5.0-1.18
		bettermounthud: Better Mount HUD 1.1.3
		betterpingdisplay: Better Ping Display 1.1.1
		borderlessmining: Borderless Mining 1.1.1+1.18
		capes: Capes 1.2.2+1.18
		cem: Custom Entity Models 0.7.0
		chickensshed: ChickensShed 1.18.1-1.3
		citresewn: CITResewn 0.8.1-1.18
		cleancut: CleanCut 1.18-4.0-fabric
		clickadv: Clickable Advancements Mod 2.1
		clientcommands: Client Commands 2.6
		clientcommands-scripting: Clientcommands Scripting 1.3
		cloth-api: Cloth API 2.0.54
		cloth-basic-math: cloth-basic-math 0.6.0
		cloth-client-events-v0: Cloth Client Events v0 2.0.54
		cloth-common-events-v1: Cloth Common Events v1 2.0.54
		cloth-config: Cloth Config v6 6.1.48
		cloth-datagen-api-v1: Cloth Datagen v1 2.0.54
		cloth-scissors-api-v1: Cloth Scissors API v1 2.0.54
		cloth-utils-v1: Cloth Utils v1 2.0.54
		colormatic: Colormatic 2.4.0+mc.1.18
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		completeconfig: CompleteConfig 1.3.3
		continuity: Continuity 1.0.3+1.18
		crowdin-translate: CrowdinTranslate 1.4+1.18
		cullleaves: Cull Leaves 2.3.2
		damagetilt: Damage Tilt 1.18-fabric-0.1.2
		dcch: DCCH 1.0
		disable_custom_worlds_advice: Disable Custom Worlds Advice 1.4
		dsurround: §6Dynamic Surroundings§r: §eFabric Edition 0.0.4
		dynamicfps: Dynamic FPS 2.0.6
		dynamicsoundfilters: Dynamic Sound Filters 1.3.0+1.18
		enhancedblockentities: Enhanced Block Entities 0.5+1.18
		fabric: Fabric API 0.44.0+1.18
		fabric-api-base: Fabric API Base 0.4.1+b4f4f6cd14
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.4.0+16d92c47c8
		fabric-biome-api-v1: Fabric Biome API (v1) 6.0.1+ded849a9c8
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.9+3ac43d95c8
		fabric-command-api-v1: Fabric Command API (v1) 1.1.6+3ac43d9514
		fabric-commands-v0: Fabric Commands (v0) 0.2.5+b4f4f6cdc8
		fabric-containers-v0: Fabric Containers (v0) 0.1.18+d154e2c6c8
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.4.5+6f53a73dc8
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.8+3ac43d95c8
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.7+43d29571c8
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.5+6b21378ac8
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.16+bfa23f17c8
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.6+b4f4f6cdc8
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.10+3ac43d95c8
		fabric-item-api-v1: Fabric Item API (v1) 1.3.0+691a79b5c8
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.3+3ac43d95c8
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.8+c8aba2f3c8
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.6+b4f4f6cdc8
		fabric-language-kotlin: Fabric Language Kotlin 1.7.1+kotlin.1.6.10
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.10+c15ca335c8
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.8+3ac43d95c8
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 1.0.3+3ac43d95c8
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.7+b4f4f6cdc8
		fabric-models-v0: Fabric Models (v0) 0.3.3+3ac43d95c8
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.18+3ac43d95c8
		fabric-networking-v0: Fabric Networking (v0) 0.3.5+b4f4f6cdc8
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.11.0+3b82842ec8
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.8+3ac43d95c8
		fabric-particles-v1: Fabric Particles (v1) 0.2.9+526dc1acc8
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.8.5+3ac43d95c8
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.9+3ac43d95c8
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.12+3ac43d95c8
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.7+b4f4f6cdc8
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.4+7242e9d7c8
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.18+3ac43d95c8
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.9+b4f4f6cdc8
		fabric-rendering-v1: Fabric Rendering (v1) 1.10.3+6b21378ac8
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.11+3ac43d9514
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.7+3ac43d95c8
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.11+3ac43d95c8
		fabric-structure-api-v1: Fabric Structure API (v1) 2.0.8+295197a7c8
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.5+3ac43d95c8
		fabric-textures-v0: Fabric Textures (v0) 1.0.9+3ac43d95c8
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.3.4+7de09f55c8
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.5+b4f4f6cdc8
		fabricloader: Fabric Loader 0.12.12
		fabrishot: Fabrishot 1.6.0
		fallingleaves: Falling Leaves 1.8.1+1.18.1
		farsight: Farsight Mod 1.18-1.5
		fastopenlinksandfolders: FastOpenResourcePacks 1.0.0
		ferritecore: FerriteCore 4.0.0
		iamverysmart: I Am Very Smart 2.1.0+mc1.18.1
		illuminations: Illuminations 1.10.2
		indium: Indium 1.0.2-alpha1+mc1.18
		inventoryprofilesnext: Inventory Profiles Next 1.2.3
		iris: Iris 1.1.3
		itemmodelfix: Item Model Fix 1.0.3+1.18
		java: Java HotSpot(TM) 64-Bit Server VM 17
		kirin: Kirin UI 1.10.0-beta.2
		lambdabettergrass: LambdaBetterGrass 1.2.2+1.17
		lambdynlights: LambDynamicLights 2.1.0+1.17
		languagereload: Language Reload 1.2
		lazydfu: LazyDFU 0.1.2
		lithium: Lithium 0.7.6
		malilib: MaLiLib 0.10.0-dev.26
		mavm: More Axolotl Variants Mod 1.0.7a
		midnightlib: MidnightLib 0.2.9
		minecraft: Minecraft 1.18.1
		minihud: MiniHUD 0.19.0-dev.20211204.230657
		mm: Manningham Mills 2.1
		modmenu: Mod Menu 3.0.1
		morechathistory: MoreChatHistory 1.1.0
		mostructures: Mo' Structures 1.3.0-pre1+1.18
		multiconnect-api: MultiConnect API 1.5
		no-telemetry: No Telemetry 1.3.0
		nofade: No Fade 1.18-2.0.1
		notenoughanimations: NotEnoughAnimations 1.4.0
		notenoughcrashes: Not Enough Crashes 4.1.3+1.18
		omega-config: OmegaConfig 1.0.8
		org_anarres_jcpp: jcpp 1.4.14
		org_codehaus_groovy_groovy: groovy 3.0.8
		org_codehaus_groovy_groovy-jsr223: groovy-jsr223 3.0.8
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.6.10
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.6.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.6.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.6.10
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.5.2
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.5.2
		org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.3.1
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.3.1
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.3.1
		org_joml_joml: joml 1.10.2
		org_openjdk_nashorn_nashorn-core: nashorn-core 15.3
		org_slf4j_slf4j-api: slf4j-api 1.7.12
		presencefootsteps: Presence Footsteps 1.4.0
		pride: Pride Lib 1.1.0+1.17
		reeses-sodium-options: Reese's Sodium Options 1.2.3
		skinlayers: 3d Skin Layers 1.3.1
		slight-gui-modifications: 'Slight' GUI Modifications 2.0.2
		smoothboot: Smooth Boot 1.18-1.7.0
		sodium: Sodium 0.4.0-alpha5+build.9
		sodium-extra: Sodium Extra 0.3.7
		spruceui: SpruceUI 3.3.2+1.17
		starlight: Starlight 1.0.0+fabric.d0a3220
		tooltipfix: ToolTip Fix 1.0.4-1.18
		torohealth: ToroHealth Damage Indicators 1.18-fabric-2
		waveycapes: WaveyCapes 1.0.4
		wi_zoom: WI Zoom 1.3-MC1.18.1
		yosbr: YOSBR 0.1.1

**Comment by Arrowpuppet** [comment]
Looks like it might actually be Lambdabettergrass, somehow a better grass mod would cause this issue. Who would've thought?
